mod-source-record-manager-server/src/main/java/org/folio/rest/impl/MetadataProviderImpl.java

```diff
@@ -18,13 +21,21 @@
 import org.folio.spring.SpringContextUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import javax.ws.rs.BadRequestException;
 import javax.ws.rs.NotFoundException;
 import javax.ws.rs.core.Response;
+import java.util.Date;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public class MetadataProviderImpl implements MetadataProvider {
 
   private static final Logger LOGGER = LogManager.getLogger();
+  private static final Set<Object> JOB_EXECUTION_SORTABLE_FIELDS =
```

Comment:
We can use Set of Strings instead of Set of Objects to make it more clear.

Reply:
yes sure, for some reason during extracting to constants type was generated in such way, thanks!